frontend/apps/filemanager/filemanager.lua

```diff
@@ -852,9 +861,13 @@ function FileManager:setHome(path)
     return true
 end
 
-function FileManager:openRandomFile(dir)
-    local match_func = function(file) -- documents, not yet opened
-        return DocumentRegistry:hasProvider(file) and not BookList.hasBookBeenOpened(file)
+function FileManager:openRandomFile(dir, is_unopened)
```

Comment:
I'd name it something like `unopened_only` or `skip_opened` or a variety of other things that don't imply it's a property of a file. Besides that lgtm.

Reply:
Thanks! I've updated the PR to rename the parameter